src/components/OfflineEvent/EventBox.tsx

```
@@ -1,20 +1,20 @@
 // components/OfflineEventCard.tsx
-import { ComponentProps } from "react";
-import { Text } from "components/@common/Text";
-import Box from "wowds-ui/Box";
-import Tag from "wowds-ui/Tag";
 import { Flex } from "@/components/@common/Flex";
 import { Space } from "@/components/@common/Space";
 import { useCountdown } from "@/hooks/contexts/useCountDownDate";
 import { EventStatus } from "@/types/entities/event";
+import { Text } from "components/@common/Text";
```

ESLint

[error] `components/@common/Text` import should occur before import of `@/components/@common/Flex`

(import/order)